interfaces/policy/policy_test.go

```diff
@@ -747,6 +778,13 @@ plugs:
   slots-arity-slot-one-plug-any:
     allow-auto-connection:
       slots-per-plug: *
+  plugs-name-bound:
+    allow-auto-connection:
+      -
+        plug-names:
+          - plugs-name-bound-p1
+        slot-names:
+          - plugs-name-bound-s2
```

Comment:
Did you mean plugs-name-bound-s*1* here? Down below in 'slots-name-bound' you used 'p2' an 's2' so thought you might've meant 'p1' and 's1' here (the tests below check for this correctly so it feels intentional).

Reply:
not particularly, just allowing one combination out of the 4